OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseTabsDashboardFragment.java

```diff
@@ -270,6 +269,13 @@ private View.OnClickListener onCloseClick() {
         return v -> requireActivity().finish();
     }
 
+    private View.OnClickListener onExploreCoursesClick() {
```

Comment:
we can use `onFindCourseClick()` instead

Reply:
I didn't want to send the analytics but I think it's acceptable now.